core/new-gui/src/environments/environment.default.ts

```diff
@@ -73,6 +73,12 @@ export const defaultEnvironment = {
   google: {
     clientID: "",
   },
+
+   /**
+    * Whether workflow collab should be active
+    */
+
+    enableWorkflowCollab: false,
```

Comment:
please fix format to align with other configs.

Reply:
also change it to `workflowCollabEnabled`

Reply:
Done.